recipes/openmm-cpu/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DBUILD_TESTING=OFF"
+
+# Ensure we build a release
+CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Release"
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    # CFLAGS
+    # JRG: Had to add -ldl to prevent linking errors (dlopen, etc)
+    MINIMAL_CFLAGS+=" -g -O3 -ldl"
+    CFLAGS+=" $MINIMAL_CFLAGS"
+    CXXFLAGS+=" $MINIMAL_CFLAGS"
```

Comment:
Dont override. Append

Reply:
Isn't `+=` the append to string operator?

Reply:
Yes, you are right